vm/src/main/scala/pravda/vm/Vm.scala

```diff
@@ -150,10 +150,12 @@ object Vm {
             } else {
               memory.push(addressToData(progAddress.get))
             }
+            aux()
```

Comment:
Looks like it was a critical bug not found yet :)

Reply:
yes. It was :( PR with these opcodes was merged a couple days before.

Reply:
:smiling_imp: side-effects :smiling_imp: